stories/Input/examples/Commands.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Markdown from 'wix-storybook-utils/dist/src/Markdown';
+
+import { Cell, Layout } from '../../../src';
+import LiveCodeExample from '../../utils/Components/LiveCodeExample';
+
+const Example = `class CommandsExample extends React.Component {
+  render() {
+    const handleClick1 = () => this.refs.inputtest.focus();
+
+    const handleClick2 = () => {
+      this.refs.inputtest.focus();
+      setTimeout(() => this.refs.inputtest.blur(), 1000);
+    };
+
+    const handleClick3 = () => {
+      this.refs.inputtest.focus();
+      this.refs.inputtest.select();
+    };
+
+    return (
+      <div>
+        <div style={{ width: '400px' }}>
+          <Input theme={this.props.theme} ref="inputtest" />
```

Comment:
```suggestion
          <Input theme={this.props.theme} ref={r => this.inputRef= r} />
```

and then use it like this
```js
const handleClick1 = () => this.inputReft.focus();
```